src/components/label/label.wp.scss

```diff
@@ -61,8 +61,8 @@ html[dir="rtl"] .label-wp[floating] {
   transform: translate3d(0, 0, 0) scale(.8);
 }
 
-html[dir="rtl"] .input-has-focus .label-wp[floating],
-html[dir="rtl"] .input-has-value .label-wp[floating] {
+[dir="rtl"] .input-has-focus .label-wp[floating],
```

Comment:
I'm not sure what you mean...
I am referring to:
```scss
.input-has-focus .label-wp[floating],
.input-has-value .label-wp[floating] {
  transform: translate3d(0, 0, 0) scale(.8);
}

[dir="rtl"] .input-has-focus .label-wp[floating],
[dir="rtl"] .input-has-value .label-wp[floating] {
  transform: translate3d(0, 0, 0) scale(.8);
}
```

Which as I see it, is equivalent to:
```scss
.input-has-focus .label-wp[floating],
.input-has-value .label-wp[floating] {
  transform: translate3d(0, 0, 0) scale(.8);
}
```

Because you override the ltr with the same behavior